CONTRIBUTING.md

```diff
@@ -1,23 +1,9 @@
-## C++ Style Guide
+# Contribute To PyTorch/XLA
 
-`pytorch/xla` uses `clang-format-7` with a customized style config.
-If your PR touches the C++ source files, please run the following command before submmiting a PR.
-
-```Shell
-# If your PR only changes foo.cpp, run the following in xla/ folder
-clang-format-7 -i -style /PATH/TO/foo.cpp
-# To format all cpp files, run the follwoing in xla/ folder
-find -name '*.cpp' -o -name '*.h' | xargs clang-format-7 -i -style=file
-```
+We appreciate all contributions. If you are planning to contribute bug fix for an open issue, please comment on the thread and we're happy to provide any guidance. You are very welcome to pick issues from `good first issue` and `help wanted` labels.
```

Comment:
I'm not sure about using back quotes for issue labels, those aren't code. Maybe just regular, double quotes?

Reply:
Also, "contribute a bug fix" sounds better than just "contribute bug fix".